README.md

```diff
@@ -279,6 +277,23 @@ The [minimist](https://www.npmjs.com/package/minimist) package.
 
 Available as global const `argv`.
 
+Run script:
+
+```bash
+$ zx test.js --name=utopia
+# $ zx test.js --name utopia
```

Comment:
we already have a link to minimist documentation. I think one example is good enough.